src/cpp/jank/runtime/obj/transient_vector.cpp

```diff
@@ -0,0 +1,172 @@
+#include <jank/runtime/detail/object_util.hpp>
+#include <jank/runtime/erasure.hpp>
+#include <jank/runtime/obj/persistent_vector_sequence.hpp>
+#include <jank/runtime/util.hpp>
+#include <jank/runtime/obj/native_function_wrapper.hpp>
+#include <jank/runtime/obj/transient_vector.hpp>
+#include <jank/type.hpp>
```

Comment:
I would figure you don't need half of these, since they're included so early in the runtime. Do you?